.github/workflows/at_server_prod_deploy.yaml

```diff
@@ -44,22 +44,26 @@ jobs:
     needs: Docker_Build
     runs-on: [self-hosted, linux, x64, K8s]
     steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
       # Extract branch for docker tag
       - name: Get branch name
         run: echo "BRANCH=${GITHUB_REF##*/}" >> $GITHUB_ENV
 
       # Setup gcloud CLI
-      - uses: google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce # v1.0.1
+      - id: 'auth'
+        uses: google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce # v1.0.1
         with:
-          service_account_key: ${{ secrets.GKE_SA_PROD_KEY }}
-          project_id: ${{ secrets.GKE_PROD_PROJECT }}
+          credentials_json: '${{ secrets.GKE_SA_PROD_KEY }}'
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce # v1.0.1'  
 
       # Get the GKE credentials so we can deploy to the cluster
-      - uses: google-github-actions/get-gke-credentials@894c221960ab1bc16a69902f29f090638cca753f # v1.0.1
+      - uses: google-github-actions/get-gke-credentials@0a5f57b989b81d2725178cda257a00acf7d6bb5f # v0.7.0
```

Comment:
Why these different versions?

v1.0.1 initially (which is presumably latest), but then v1.0.0 for dev and v0.7.0 for prod

Reply:
Fixed version issue